src/components/layout/nav/search/Search.jsx

```diff
@@ -0,0 +1,157 @@
+import React, { useState } from "react";
+import Col from "react-bootstrap/Col";
+import Button from "react-bootstrap/Button";
+import Form from "react-bootstrap/Form";
+import "./Search.scss";
+import useDeviceSizes from "../../useDeviceSizes";
+
+//This will be serving as a temporary database until we build the real one completely.
+export const mockDatabase = [
```

Comment:
this is duplicated with omar's code. you should only have it in one place. please work with him